downstairs/src/lib.rs

```diff
@@ -3237,6 +3162,111 @@ impl Downstairs {
         }
         Ok(())
     }
+
+    /// Binds the given `ConnectionId` to a new connection state
+    ///
+    /// # Panics
```

Comment:
Panics when there is no repair_address or if a previous connection state
exists at the same ID?

Reply:
yup, done in 22f4e4538bcd8719fe7b23ab113c02c3439f39e1